packages/graphql-language-service-server/src/parseDocument.ts

```diff
@@ -48,7 +49,7 @@ export function parseDocument(
   uri: string,
   fileExtensions: string[] = DEFAULT_SUPPORTED_EXTENSIONS,
   graphQLFileExtensions: string[] = DEFAULT_SUPPORTED_GRAPHQL_EXTENSIONS,
-  logger: Logger = new Logger(),
+  logger: Logger = new ConsoleLogger(),
```

Comment:
I don't think we even need to instantiate a logger as a default here, not sure why it was added to begin with! again i think this should be fixed in the test suite

Reply:
The default value is there due to tests calling `parseDocument` with only two args since `fileExtensions` and `graphQLFileExtensions` have default values.  I added a simple noop logger for tests.